docs/guide/blog.md

```diff
@@ -196,7 +253,7 @@ func (k Keeper) SetPostCount(ctx sdk.Context, count uint64) {
 }
 ```
 
-Now that we've implemented functions for getting the number of posts and setting the post count, we can implement the logic behind `AppendPost`:
+Now that you have implemented functions for getting the number of posts and setting the post count, you can implement the logic behind `AppendPost`. The `import` section is also shown in this example:
```

Comment:
is this the proper location to show the imports and is this list complete? I'm a little murky on why some files have smaller import sets?

Reply:
```suggestion
Now that you have implemented functions for getting the number of posts and setting the post count, you can implement the logic behind `AppendPost`.:
```